airflow/models/dag.py

```diff
@@ -1797,20 +1797,6 @@ def get_paused_dag_ids(dag_ids: List[str], session: Session = None) -> Set[str]:
     def safe_dag_id(self):
         return self.dag_id.replace('.', '__dot__')
 
-    def get_dag(self, store_serialized_dags=False):
-        """Creates a dagbag to load and return a DAG.
-
-        Calling it from UI should set store_serialized_dags = STORE_SERIALIZED_DAGS.
-        There may be a delay for scheduler to write serialized DAG into database,
-        loads from file in this case.
-        FIXME: remove it when webserver does not access to DAG folder in future.
-        """
-        dag = DagBag(
```

Comment:
We can't use the DagBag class in the airflow.models.dag module.

Reply:
Removing this method broke `DagModel.create_dagrun` which broke Trigger Dag View.

PR to solve this: https://github.com/apache/airflow/pull/8148